### DISCUSSION TITLE
suggestion: sort dependents by stars

### DISCUSSION THREAD

**Discussion by wkalt** [community/community#84073] [Repositories]
### Select Topic Area

Product Feedback

### Body

If I am looking at this page:
https://github.com/typeorm/typeorm/network/dependents

what I mostly want to know is "what big projects use this package". If the list were ordered by the star count that would give it to me. But I don't see a way to make that happen.

What do you think?

**Comment by adamdecaf** [comment]
Yes! I was asking Github Support if there's an API for this, so I could sort it when the UI doesn't support sorting.
